### DISCUSSION TITLE
Workspace not found while running lookup table method

### DISCUSSION THREAD

**Discussion by KeshavPeri** [cloud-carbon-footprint/cloud-carbon-footprint#1246] [Q&A]
Hi All. Would appreciate some urgent help on this query.

I am trying to run the application via the create lookup table method. When i run the create-look-up table command i get the following error listed in the screenshot below
![Screenshot 2023-10-02 071734](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/assets/66486403/8ca1764a-7b9d-458c-baed-fde7ed381ca1)

**Comment by camcash17** [comment]
Hi @KeshavPeri, could you please provide more information about the steps you followed downloading/installing the application? Please provide any other useful screenshots of this process.
